target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -2013,6 +2013,9 @@ static int rtl83xx_port_pre_bridge_flags(struct dsa_switch *ds, int port, struct
 		features |= BR_MCAST_FLOOD;
 	if (priv->r->enable_bcast_flood)
 		features |= BR_BCAST_FLOOD;
+	if (priv->r->port_isolation)
+		features |= BR_ISOLATED;
+
```

Comment:
Not happy about the commit message.

But the commit message also states 

> Care is also take that the feature is only enabled for the intended chips
> and other chips are unaffected.

But still, this is activated without any check

Reply:
Then explain it in the commit + PR message